mediator/esmFldsExchange_nems_mod.F90

```diff
@@ -753,13 +791,11 @@ subroutine esmFldsExchange_nems(gcomp, phase, rc)
 
     ! to lnd - states and fluxes from atm
     if ( trim(coupling_mode) == 'nems_orig_data') then
-       allocate(flds(21))
+       allocate(flds(16))
        flds = (/'Sa_z      ', 'Sa_topo   ', 'Sa_tbot   ', 'Sa_pbot   ', &
```

Comment:
I don't see Sa_topo as a field being sent toLnd in the NoahMP cap. Is this field required?

Reply:
@DeniseWorthen I think it is not currently used but we might use it to apply topography corrections to temperature like following, https://github.com/NOAA-EMC/ufs-land-driver/blob/972bac9dcfb187fc00af05770cabf99dacdaf294/driver/ufsLandForcingModule.f90#L493. I could remove it for now.

Reply:
@DeniseWorthen We discussed it in our regular meeting yesterday.  If you don't mind, we could keep as it is. In any case we will have another round of PR to bring restructured land component code and that could also include some cleaning in the interfaces. The document that we wrote for the coupling interface mainly reflects the current version of the component model. Since this PR is not merged yet (I am not sure when ESMF 8.5.0 will be available and PR will in) I did not update the document.